plugin/src/main/kotlin/com/ritense/plugin/service/PluginService.kt

```diff
@@ -165,12 +167,12 @@ class PluginService(
         pluginProcessLinkRepository.save(link)
     }
 
-    fun invoke(execution: DelegateExecution, processLink: PluginProcessLink) {
+    fun invoke(execution: DelegateExecution, processLink: PluginProcessLink): Any? {
         val instance = createInstance(processLink.pluginConfigurationId)
 
         val method = getActionMethod(instance, processLink)
         val methodArguments = resolveMethodArguments(method, execution, processLink.actionProperties)
-        method.invoke(instance, *methodArguments)
+        return method.invoke(instance, *methodArguments)
```

Comment:
Do we have a use case (other than testing) for returning a value on an action? We can't use the returned the way an action is configured right now, can we?

Reply:
When invoking an action through code. So not as important for the development team, but it's useful for implementation teams.